### DISCUSSION TITLE
Github pages Custom domain not working

### DISCUSSION THREAD

**Discussion by Emmanuel-Aggrey** [community/community#35489] [Other Feature Feedback, Questions, & Ideas]
The custom domain `selannsproperties.com` is already taken. If you are the owner of this domain, check out https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages for information about how to verify and release this domain.

I have verified it with the TXT record from the domain DNS settings but still getting this message. I can not find the repository that is causing this either.

![image](https://user-images.githubusercontent.com/39681748/194680101-a91ffe61-fe1a-4d46-a8a4-357dcca37820.png)

**Comment by anjanesh** [comment]
Just point the A records to :
```
185.199.108.153
185.199.109.153
185.199.110.153
185.199.111.153
```

**Comment by ahopkins** [comment]
I have this same problem. DNS is setup correctly. Domain is verified. Same issue persists.

![image](https://user-images.githubusercontent.com/166269/195156409-2eb6daba-47af-4004-9fc0-63e8ca2fedb2.png)
![image](https://user-images.githubusercontent.com/166269/195156482-e995dcc2-9b66-46b5-8be8-90d6fc9aed40.png)
